app/assets/javascripts/discourse/app/lib/constants.js

```diff
@@ -89,7 +89,6 @@ export const SYSTEM_FLAG_IDS = {
   off_topic: 3,
   inappropriate: 4,
   spam: 8,
-  illegal: 10,
```

Comment:
Is there a reason for removing this constant?

Reply:
Whoops. The task added some gibberish and I accidentally removed too much. 🙏